stories/wmts.planet.stories.js

```diff
@@ -37,7 +37,6 @@ export const getMapBbox = () => {
       width: 256,
       height: 256,
       format: MimeTypes.JPEG,
-      zoom: 14,
```

Comment:
If I understand this correctly, there are 2 ways of setting the "location" 
- bbox (and no zoom is required)
- tileCoord

The **bbox and no zoom** conclusion is based on no `zoom` being present in `src/layer/WmtsLayer.ts`.
If this conclusion is correct, should the `zoom` also be removed from:
- `GetMapParams` in `src/layer/const.ts`
- `WmtsLayer.getMap uses bbox` test in `src/layer/__tests__/WmtsLayer.ts`
-

Reply:
Fixed. Thanks for the thorough check.

Reply:
Looks good now. Resolving